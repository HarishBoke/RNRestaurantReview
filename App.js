/**
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import {SafeAreaView, View, Text, StatusBar} from 'react-native';
import styles from './src/assets/styles/style.style';

import AppHeader from './src/components/AppHeader';
import Restaurant from './src/scenes/Restaurant';

const App: () => React$Node = () => (
  <>
    {/* take care on notch and topbar visible , full height*/}
    <StatusBar barStyle="dark-content" />
    <SafeAreaView>
      <AppHeader />
      {global.HermesInternal == null ? null : (
        <View style={styles.engine}>
          <Text style={styles.footer}>Engine: Hermes</Text>
        </View>
      )}
      <Restaurant />
    </SafeAreaView>
  </>
);

export default App;
