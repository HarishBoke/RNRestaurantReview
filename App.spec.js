import React from 'react';
import {shallow} from 'enzyme';

import App from './App';

const initApp = () => {
  const wrapper = shallow(<App />);
  return {wrapper};
};

describe('Component: App', () => {
  describe('rendering', () => {
    it('should display the application', async () => {
      const {wrapper} = initApp();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
