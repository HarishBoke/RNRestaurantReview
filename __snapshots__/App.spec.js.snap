// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App rendering should display the application 1`] = `ShallowWrapper {}`;
